doc/host-setup.md

```diff
@@ -146,3 +146,39 @@ Checking the current value again will reveal that the helper is now disabled
 # cat /sys/module/drbd/parameters/usermode_helper
 disabled
 ```
+
+## Disable Lvmetad in CentOS 7 and Ubuntu 18.04
+
+Reference: https://access.redhat.com/solutions/2053483
+
+CentOS 7 and Ubuntu 18.04 by default use **lvmetad**,  a metadata caching daemon for LVM. The daemon receives notifications from udev rules. Through these notifications, lvmetad has an up-to-date and consistent image of the volume groups available in the system. lvmetad improves performance of LVM commands by saving them from scanning the disks in every execution. 
+
+`piraeus-ns-node` pod runs lvm commands inside a container without using any lvmetad or udev. And it does not communicate to lvmetad daemon in the operation system. Therefore, users must run `pvscan --cache` in the OS each time to view LVM changes after a Piraeus volume operation. 
+
+In CentOS 8 and in Ubuntu 20.04 (focal), lvmetad is deprecated because lvm2 has removed it since version 2.03. **Therefore, it is also recommended to disable lvmetad in CentOS 7 and Ubuntu 18.04 binoic**. 
+
+Follow below steps to disable lvmetad completely: 
+```bash
+$ systemctl stop lvm2-lvmetad.socket lvm2-lvmetad.service
+
+$ systemctl disable lvm2-lvmetad.service
+
+$ systemctl mask lvm2-lvmetad.socket
+
+$ sed -i 's/use_lvmetad = 1/use_lvmetad = 0/' /etc/lvm/lvm.conf
+
+$ cat /etc/lvm/lvm.conf | grep use_lvmetad
+use_lvmetad = 0
+```
+If lvm is used for root, initial RAM has to be updated:
```

Comment:
Is it really necessary step? You already disabled `lvm2-lvmetad.service` and `lvm2-lvmetad.socket`, and configured `lvm.conf` to not use it.

Why do we need any changes on initramfs phase?

Reply:
It is recommended by Redhat to load lvm.conf properly during boot, specifically for root on lvm. Apart from that, in CentOS 7 without updated intramfs, commends such as vgdisplay will still complain about missing lvmetad.